pymatbridge/matlab_magic.py

```diff
@@ -211,6 +211,8 @@ def matlab(self, line, cell=None, local_ns=None):
             else:
                 e_s = "There was an error running the code:\n %s"%code
                 result_dict = self.eval(code)
+        except MatlabInterperterError:
+            raise
         except:
```

Comment:
I don't know enough about your code to say. I'm guessing that some other kind of exception is raised when a MATLAB session cannot be found and it would be caught here.

However, I agree that it should be removed if possible, it's not great that it may catch and hide other exceptions at this point to. Do you know what exception gets raised when the interpreter can't be found? If there isn't one, maybe one can be added and this problem could be avoided.